src/components/MovieList.js

```diff
@@ -0,0 +1,95 @@
+import styled from 'styled-components';
+
+import useStore from './../hooks/useStore';
+import Button from './Button';
+
+const Section = styled.section`
+	padding: 16px;
+`;
+
+const Article = styled.article`
+	max-width: 154px;
+	padding: 0;
+	animation: fadeIn 1s ${({delay}) => delay}s forwards;
+	border: 1px solid rgba(0, 0, 0, 0.2);
+	border-radius: 20px;
+	opacity: 0;
+	background-color: #fff;
+	box-shadow: 1px 1px 10px 5px rgba(0, 0, 0, 0.1);
+	color: #fff;
+
+	@keyframes fadeIn {
+		0% {
+			opacity: 0;
+		}
+		100% {
+			opacity: 1;
+		}
+	}
+`;
+
+const Picture = styled.img`
+	border-radius: 20px 20px 0 0;
+`;
+
+const Grid = styled.div`
+	display: flex;
+	flex-wrap: wrap;
+	gap: 16px;
+`;
+
+const Movieheadline = styled.p`
+	padding: 8px 8px 16px 8px;
+	color: black;
+	overflow-wrap: break-word;
+`;
+
+export default function MovieList() {
+	const arrData = useStore(state => state.arrData);
+	const setData = useStore(state => state.setData);
+	const arrConfiguration = useStore(state => state.arrConfiguration);
```

Comment:
Please do not use abbreviations. Variable names should always be descriptive. I think you can find better names here.

Reply:
Those are not abbreviations. I'm using arrCamelCaseName because it it an Array thats what the "arr" as a prefix stands for. If it would be an Object or a string, it would be objData or strConfiguration. Booleans can go with "is" or "has", like you've seen in my code.
This is a common practice, especially with languages that are not type secure like JavaScript.

Reply:
I know but "arr" is clearly an abbreviation of "array". Same as we want to write "number" instead of "num", I suggest to write "array" instead of "arr" as it is much easier to read and more descriptive. Readability is more important than getting rid of some characters  🤷🏼‍♀️ In the end it's also the question if you really need to mention here that this is an array.

Reply:
I'm using this coding style since 1995 or something. So, no.